Makefile

```
@@ -14,22 +14,30 @@ endif
 VERSION ?= 1.1.1h
 
 ## Extra version of the distributed package
-PACKAGE_VERSION ?= 1
+PACKAGE_VERSION ?= 3
```

The previous one was 1.1.10801, so the next one should be 1.1.10802, I think.

```suggestion
PACKAGE_VERSION ?= 2
```

Please disregard any demo versions from my private repo.

---

I suggested having 1.1.1083 to be consistent with your private repo (because otherwise we might get confused)